client/styles/abstracts/_placeholders.scss

```diff
@@ -154,45 +190,52 @@
 		border: 1px solid getThemifyVariable('modal-border-color');
 		box-shadow: 0 12px 12px getThemifyVariable('shadow-color');
 	}
+
 	border-radius: 2px;
 	z-index: 20;
 }
 
 %hidden-element {
-	position:absolute;
-	left:-10000px;
-	top:auto;
-	width:1px;
-	height:1px;
-	overflow:hidden;
+	position: absolute;
+	left: -10000px;
+	top: auto;
+	width: 1px;
+	height: 1px;
+	overflow: hidden;
 }
 
 
 %link {
 	@include themify() {
 		text-decoration: none;
-		color: getThemifyVariable('inactive-text-color');
+		color: getThemifyVariable('inactive-text-color') !important;
 		cursor: pointer;
-		& g, & path {
-			fill: getThemifyVariable('inactive-text-color');
+
+		& g,
+		& path {
+			fill: getThemifyVariable('inactive-text-color') !important;
```

Comment:
I would remove the "!important" for the path property, since it's affecting the color scheme for the play/pause button!